aws/resource_aws_lambda_alias_test.go

```diff
@@ -78,6 +79,35 @@ func TestAccAWSLambdaAlias_nameupdate(t *testing.T) {
 	})
 }
 
+func TestAccAWSLambdaAlias_routingconfig(t *testing.T) {
```

Comment:
This acceptance test is currently failing for me:

```
make testacc TEST=./aws TESTARGS='-run=TestAccAWSLambdaAlias_routingconfig'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -run=TestAccAWSLambdaAlias_routingconfig -timeout 120m
=== RUN   TestAccAWSLambdaAlias_routingconfig
--- FAIL: TestAccAWSLambdaAlias_routingconfig (86.12s)
	testing.go:518: Step 0 error: Error applying: 1 error(s) occurred:

		* aws_lambda_alias.lambda_alias_test: 1 error(s) occurred:

		* aws_lambda_alias.lambda_alias_test: Error creating Lambda alias: ResourceNotFoundException: Function not found: arn:aws:lambda:us-west-2:187416307283:function:tf_acc_lambda_func_alias_basic_vmhu4b4h:2
			status code: 404, request id: 8e743c6f-7e17-11e8-81bb-0f6cecc9e91a
FAIL
FAIL	github.com/terraform-providers/terraform-provider-aws/aws	86.161s
```

Its complaining because version 2 of the Lambda function (newly created in the test) does not exist. I'm not sure its possible to create this configuration without at least two versions of the Lambda function existing already. When trying to set the routing configuration in the existing test configuration to:

```hcl
  routing_config   = {
    additional_version_weights = {
      "1" = 0.5
    }
  }
```

I get:

```
		* aws_lambda_alias.lambda_alias_test: Error creating Lambda alias: InvalidParameterValueException: Invalid function version 1. Function version 1 is already included in routing configuration.
			status code: 400, request id: 6f7ccbcf-7e16-11e8-b729-25d25f4dd472
```

Reply:
[This test](https://github.com/eggsbenjamin/terraform-provider-aws/blob/6086430ea2d13b8fcac3b7380c3834dbd48f224e/aws/resource_aws_lambda_alias_test.go#L43) got lost when I merged sorry. It creates multiple versions in order for traffic shifting to be possible and tests that the routing config can be added an removed.